src/test/java/com/twilio/oai/TwilioGeneratorTest.java

```diff
@@ -47,18 +41,30 @@ public static void setUp() {
 
     @Test
     public void launchGenerator() {
-        final CodegenConfigurator configurator = new CodegenConfigurator()
-            .setGeneratorName(generator.getValue())
-            .setInputSpec("examples/spec/twilio_api_v2010.yaml")
-            .setOutputDir("codegen/" + generator.getValue())
-            .setInlineSchemaNameDefaults(Map.of("arrayItemSuffix", ""))
-            .addGlobalProperty("apiTests", "false")
-            .addGlobalProperty("apiDocs", "false");
```

Comment:
same comment as above here.

Reply:
These are the changes, Changed the test script to generate code for both file and directory